specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json

```diff
@@ -8819,8 +8820,8 @@
         "x-ms-examples": {
           "Task reactivate": { "$ref": "./examples/TaskReactivate.json" }
         },
-        "summary": "Reactivates the specified task.",
-        "description": "Reactivation makes a task eligible to be retried again up to its maximum retry count. The task's state is changed to active. As the task is no longer in the completed state, any previous exit code or scheduling error is no longer available after reactivation. This will fail for tasks that are not completed or that previously completed successfully (with an exit code of 0). Additionally, this will fail if the job has completed (or is terminating or deleting).",
+        "summary": "Reactivates a task, allowing it to run again even if its retry count has been exhausted.",
+        "description": "Reactivation makes a task eligible to be retried again up to its maximum retry count. The task's state is changed to active. As the task is no longer in the completed state, any previous exit code or failure information is no longer available after reactivation. Each time a task is reactivated, its retry count is reset to 0. This will fail for tasks that are not completed or that previously completed successfully (with an exit code of 0). Additionally, this will fail if the job has completed (or is terminating or deleting).",
```

Comment:
Or "this operation" would also work

---
In reply to: [133617601](https://github.com/Azure/azure-rest-api-specs/pull/1549#discussion_r133617601) [](ancestors = 133617601)